Source/WebInspectorUI/UserInterface/Views/SingleSidebar.js

```diff
@@ -72,7 +73,7 @@ WI.SingleSidebar = class SingleSidebar extends WI.Sidebar
 
         this._recalculateWidth(newWidth);
     }
-
+    
```

Comment:
Please remove empty accidental new lines and indentation changes.

Reply:
Fixed.